src/main/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryController.java

```diff
@@ -454,6 +749,27 @@ static MediaType determineExportResponseContentType(@Nullable final ExportFormat
             description = "Change the workflow state of the advisory with the given id to Draft.",
             tags = {"Advisory"}
     )
+    @ApiResponses(value = {
+        @ApiResponse(
+          responseCode = "200", 
+          description = "Workflow state changed to draft.",
+          content = {
+              @Content(mediaType = MediaType.TEXT_PLAIN_VALUE)
+          }
+        ),
+        @ApiResponse(
+            responseCode = "422", 
+            description = "Invalid formated advisory." 
+          ),
+        @ApiResponse(
+            responseCode = "400", 
+            description = "Advisory ID not found." 
+          ),
+        @ApiResponse(
+          responseCode = "401", 
+          description = "Unauthorized access to change workflow state."
+        )
+      })
```

Comment:
Is it only `formatted` or is the advisory invalid itself?
```suggestion
        @ApiResponse(
            responseCode = "400", 
            description = "Advisory ID not found." 
          ),
        @ApiResponse(
            responseCode = "401", 
            description = "Unauthorized access to change workflow state."
        ),
        @ApiResponse(
            responseCode = "422", 
            description = "Invalid formatted advisory." 
        )
      })